Makefile-ponyc

```diff
@@ -616,7 +616,8 @@ libponyc.benchmarks.buildoptions += -DLLVM_BUILD_MODE=$(LLVM_BUILD_MODE)
 
 libgbenchmark.buildoptions := \
   -Wshadow -pedantic -pedantic-errors \
-  -Wfloat-equal -fstrict-aliasing -Wstrict-aliasing -Wno-invalid-offsetof \
+  -fstrict-aliasing -Wstrict-aliasing \
+  -Wno-invalid-offsetof -Wno-deprecated-declarations \
```

Comment:
The change here is to address what exactly?

Reply:
Hi! These were intentional.

As the commit message says, these reflect the [changes in gbenchmark's CMakeLists.txt](https://github.com/ponylang/ponyc/pull/3386/commits/eaf765cd917514eda4bf69fcde58f7c003684783#diff-434edcf37b69565204fb160bc9bbdd4b).

I think they removed `-Wfloat-equal` because if you have `-Wall`, you don't have to explicitly enable this warning. Either that or they really want to `==` floats and know what they're doing.

And `-Wno-deprecated-declarations` was added because they deprecated the `CSVExporter` and they use `-Werror` like `ponyc` does so it becomes impossible to build `gbenchmark` without disabling the warning. See https://github.com/google/benchmark/issues/608

`-Wno-invalid-offsetof` was moved to avoid mixing the `-Wno-` with other flags. This could probably be removed but it was here from the beginning so I kept it.